tools/checkstyle/checkStyle.xml

```diff
@@ -81,6 +81,14 @@
     </module>
 
     <module name="TreeWalker">
+        <module name="CustomImportOrder">
+            <property name="customImportOrderRules"
+                      value="THIRD_PARTY_PACKAGE###STANDARD_JAVA_PACKAGE###SPECIAL_IMPORTS###STATIC"/>
+            <property name="standardPackageRegExp" value="^java\."/>
+            <property name="specialImportsRegExp" value="^javax\."/>
+            <property name="sortImportsInGroupAlphabetically" value="true"/>
+            <property name="separateLineBetweenGroups" value="false"/>
+        </module>
```

Comment:
Hi @leo65535, thanks for your comment, let me try this out

Reply:
@leo65535 This is great, also I make two improvements for it, as follows:

1. Use `/^java\./` instand of `java` to ensure the accuracy;
2. Add `staticGroups` as well, otherwise this check plugin will never be satisfying.